src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Winning {
+
+    private static final String DELIMITER = ",";
+
+    private final Lotto lotto;
+
+    public Winning(String input) {
+        List<Integer> numbers = parseLottoNumbers(input);
+        this.lotto = new Lotto(numbers);
+    }
+
+    public Lotto getLotto() {
+        return lotto;
+    }
+
+    private List<Integer> parseLottoNumbers(String input) {
+        String[] inputSplit = input.split(DELIMITER);
+        return Arrays.stream(inputSplit)
+                .map(Integer::parseInt)
+                .toList();
```

Comment:
```suggestion
        return Arrays.stream(input.split(DELIMITER))
                .map(Integer::parseInt)
                .toList();
```
한 번만 이용하는 변수인데 인라인으로 리팩토링 하는건 어떨까요?

Reply:
좋은 방향성 제안 감사합니다! ☺️